### DISCUSSION TITLE
posts/%E6%97%A0%E6%89%8B%E5%81%9A%E9%A2%98%E5%AE%B6%E5%BC%82%E9%97%BB%E5%BD%95/main/

### DISCUSSION THREAD

**Discussion by giscus** [Flying-dragon-boxing/BlogDiscussions#3] [Announcements]
# posts/%E6%97%A0%E6%89%8B%E5%81%9A%E9%A2%98%E5%AE%B6%E5%BC%82%E9%97%BB%E5%BD%95/main/

本文为虚构故事，如有雷同纯属巧合 致敬传奇树洞用户🥵0️⃣🈹🈹和给给朋友 #4087001 1年前 2022-09-19 17:36 手院绩点排名挺好奇的 #18242322 1年前 2022-09-19 17:36 [Alice] 😅 #18242351 1年前 2022-09-19 17:37 [Bob] 细说 #18242375 1

https://blog.rhx0820.xyz/posts/%E6%97%A0%E6%89%8B%E5%81%9A%E9%A2%98%E5%AE%B6%E5%BC%82%E9%97%BB%E5%BD%95/main/

**Comment by GCC314** [comment]
🥵0️⃣元年！

**Reply by Flying-dragon-boxing** [reply]
猛1️⃣弟弟🥵🥵🥵
